tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/AbstractPortalBundle.java

```diff
@@ -309,6 +318,38 @@ else if (configType.equals(_CONFIG_TYPE_SERVER)) {
 		return configInfoPath;
 	}
 
+	private TimeZone _getPortalDefaultTimeZone() {
+		IPath portalLibPath = getAppServerPortalDir().append("WEB-INF/lib");
+
+		if (FileUtil.exists(portalLibPath)) {
+			IPath portalImplJarPath = portalLibPath.append("portal-impl.jar");
+
+			if (FileUtil.exists(portalImplJarPath)) {
+				try (ZipFile portalImplJar = new ZipFile(portalImplJarPath.toFile());
+					InputStream portalPropertiesInputstream = portalImplJar.getInputStream(
```

Comment:
What about portal-ext.propertes?

Reply:
if user has been defined portal-ext properties file, can we can use the log file name as default format？ Its possible user define a customized file name like append additional suffix steing. That means we have to parse the log4j file and analyze it. maybe its not worth to do this, I prefer we only support default setting and we can give a warning while we could not find liferay log

Reply:
I feel we just should notify user that the log we just support default log file path is enough. Do you have any idea about it?